tomee/tomee-webaccess/pom.xml

```diff
@@ -42,6 +42,19 @@
         <activeByDefault>true</activeByDefault>
       </activation>
     </profile>
+    <profile>
+      <id>jdk15+</id>
+      <activation>
+        <jdk>[15,)</jdk>
```

Comment:
imo no reason to hide this dependency inside of a profile if the compile level is raised to 17 (also applies to java11plus profile above)